core/storage/opportunity/gae_models.py

```diff
@@ -126,9 +129,15 @@ def get_all_translation_opportunities(
         else:
             start_cursor = datastore_services.make_cursor()
 
-        language_query = cls.query(
-            cls.incomplete_translation_language_codes == language_code
-        ).order(cls.incomplete_translation_language_codes)
+        if not topic_name:
+            language_query = cls.query(
```

Comment:
Since this part of the query remains the same, can we create a base query with these parameters and just call `.filter(cls.topic_name == topic_name)` if `topic_name`?

Reply:
I am not sure how to create a base query when the arguments of `cls.query()` are different. Any suggestions?